README.md

```diff
@@ -7,10 +7,11 @@ iocage is a jail/container manager amalgamating some of the best features and
 technologies the FreeBSD operating system has to offer. It is geared for ease
  of use with a simple and easy to understand command syntax.
 
-iocage is in the FreeBSD ports tree as sysutils/iocage.
-To install using binary packages, simply run: `pkg install iocage`
+iocage is in the FreeBSD ports tree as sysutils/py-iocage.
+To install using binary packages, simply run: `sysutils/py-iocage`
```

Comment:
@skarekrow,

Yeah, what's up with that. https://www.freshports.org/sysutils/py-iocage but says pkg install py27-iocage

**doesn't exist** https://svnweb.freebsd.org/ports/head/sysutils/py27-iocage 
**exists** https://svnweb.freebsd.org/ports/head/sysutils/py-iocage